src/tests/intg/test_pam_responder.py

```diff
@@ -39,10 +39,6 @@
 LDAP_BASE_DN = "dc=example,dc=com"
 
 
-def have_files_provider():
```

Comment:
Ack. I just wonder if it would make sense to run the tests against both files and proxy provider since files provider has not yet been removed from SSSD? But perhaps it is not worth the effort.

Reply:
Hi,

while running test with both providers would be easy with parametrized fixtures I'm currently not sure what would be the best/most pythonic way to make the parameter list conditional.

@alexey-tikhonov, do you think we should better run both test as long as the files provider is build?

bye,
Sumit

Reply:
I think you need to use indirect parametrization, eg https://stackoverflow.com/questions/42228895/how-to-parametrize-a-pytest-fixture and check request.param in each fixture that deals with configuration.

But perhaps the easiest way would be to just copy and paste the tests.

But maybe it is not worth the effort.

Reply:
It if makes things easier, we could run tests against 'files provider' on centos-8 and centos-9, and against 'proxy' on all other platforms (including centos-10 soon), i.e. never both on a given target...

If it's troublesome, then running against 'proxy' on all targets should be also ok because underlying certmap lib is the same.

Reply:
Hi,

the new version includes parametrized tests.

bye,
Sumit